src/sidebar/services/test/streamer-test.js

```diff
@@ -211,7 +211,7 @@ describe('StreamerService', () => {
       return activeStreamer.connect().then(() => {
         assert.equal(
           fakeWebSocket.url,
-          'ws://example.com/ws?access_token=dummy-access-token&foo=bar'
+          'ws://example.com/ws?foo=bar&access_token=dummy-access-token'
```

Comment:
Can you shed light on this change?

Reply:
The sort order of the original params is preserved and the added `access_token` param is added at the end.